images/policy-bundle.json

```diff
@@ -49,6 +49,464 @@
           "trigger_id": "CVE-2019-11324+python3-pip"
         }
       ]
+    },
+    {
+      "id": "whitelist_ignore_rhsa_feed_on_centos",
+      "name": "Whitelist ignore RHSA feed on centos",
+      "version": "1_0",
+      "comment": "Work around https://docs.anchore.com/current/docs/releasenotes/2.3.0/centos_to_rhel_upgrade/",
```

Comment:
wait, so anchore is now marking things in centos that have patches in redhat as unpatched but available?

Reply:
yes